src/analysis/inner_product_space/two_dim.lean

```diff
@@ -473,4 +535,82 @@ begin
   simp,
 end
 
+lemma kahler_map {F : Type*} [inner_product_space ℝ F] [fact (finrank ℝ F = 2)]
+  (φ : E ≃ₗᵢ[ℝ] F) (x y : F) :
+  (orientation.map (fin 2) φ.to_linear_equiv o).kahler x y = o.kahler (φ.symm x) (φ.symm y) :=
+by simp [kahler_apply_apply, area_form_map]
```

Comment:
for most other lemmas, you add applied versions assuming you have a linear isometry with positive determinant. Would it be worth adding it here too?